### PR TITLE
MRG, BUG: Allow depth > 1

### PR DESCRIPTION
**Pull request by larsoner**
From discourse

### CODE REVIEW

File: mne/minimum_norm/inverse.py [hunk 1]
```diff
@@ -1372,9 +1372,9 @@ def _prepare_forward(forward, info, noise_cov, fixed, loose, rank, pca,
     # Deal with "depth"
     if exp is not None:
         exp = float(exp)
-        if not (0 <= exp <= 1):
-            raise ValueError('depth exponent should be a scalar between '
-                             '0 and 1, got %s' % (exp,))
+        if not 0 <= exp:
+            raise ValueError('depth exponent should be a greater than or '
```
Comment (hoechenberger) [inline mne/minimum_norm/inverse.py:1376]:
```suggestion
            raise ValueError('depth exponent should be greater than or '
```

File: mne/minimum_norm/inverse.py [hunk 2]
```diff
@@ -1372,9 +1372,9 @@ def _prepare_forward(forward, info, noise_cov, fixed, loose, rank, pca,
...
+                             'equal to 0, got %s' % (exp,))
```
Comment (hoechenberger) [inline mne/minimum_norm/inverse.py:1377]: wanna turn this into an f-string? :)

**Review by hoechenberger — APPROVED** [review]
two nitpicks, otherwise lgtm

File: mne/minimum_norm/inverse.py [hunk 3]
```diff
@@ -1372,9 +1372,9 @@ def _prepare_forward(forward, info, noise_cov, fixed, loose, rank, pca,
     # Deal with "depth"
     if exp is not None:
         exp = float(exp)
-        if not (0 <= exp <= 1):
-            raise ValueError('depth exponent should be a scalar between '
-                             '0 and 1, got %s' % (exp,))
+        if not 0 <= exp:
```
Comment (agramfort) [inline mne/minimum_norm/inverse.py:1375]:
```suggestion
        if exp < 0:
```

**Review by agramfort — COMMENTED** [review]
besides LGTM

thx @larsoner

**Pull request merged**